src/components/views/homePage.tsx

```diff
@@ -0,0 +1,55 @@
+import { push } from 'connected-react-router';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import * as Redux from 'redux';
+import { RootAction } from '../../actions';
+import { getActiveSong, getAllSongsIDS, getNextSongID, getPreviousSongID } from '../../reducers/songs';
+import { AppState, SongState } from '../../types/store';
+import { Browser } from '../browser';
+import { Navbar } from '../navbar';
+import { Player } from '../player';
+import { Row } from '../utils/style';
+
+const Home = (props: HomeProps) => {
+  const {songsIDs, activeSong, nextSongID, previousSongID, playSong} = props;
+  return (
+    <div>
+      <Navbar/>
+      <Row>
+        <Browser songsIDs={songsIDs}/>
+        <Player currentSong={activeSong} previousSongID={previousSongID} nextSongID={nextSongID} playSong={playSong}/>
```

Comment:
Why aren't all of these props taken from the store (in `Player` instead of here)?